charts/stable/whisparr/Chart.yaml

```
@@ -22,7 +22,7 @@ sources:
   - https://github.com/truecharts/charts/tree/master/charts/stable/whisparr
   - https://github.com/onedr0p/containers/tree/main/apps/whisparr
 type: application
-version: 6.0.0
+version: 6.0.1
```

```suggestion
version: 6.1.0
```

Feat is always minor version increase!

---

Noted